[38;5;226m[line 4] Warning after 'return': Unreachable code.[0m
0 error(s) and 1 warning(s) occurred
init
Foo instance
